godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -25,7 +25,7 @@ class TractViewController: BaseViewController {
         super.viewDidLoad()
         
         getData()
-        displayTitle()
+        displayScreenTitle()
```

Comment:
You might not need to call this any more now that we are just overriding a superclass method to set the title element. This is already called in BaseViewController.viewWillAppear().

I originally took a more manual approach to setting the custom title on this view, but then refactored it to use the pattern you created in the view controller. I'm pretty sure this line can be removed.